rel/files/mongooseim.cfg

```diff
@@ -122,14 +122,20 @@
       {transport_options, [{max_connections, 1024}, {num_acceptors, 10}]},
       {modules, [
 
-          {"_", "/http-bind", mod_bosh},
```

Comment:
Any thoughts on how you want it written for TOML? The precedent in `mod_revproxy` is tuple list of headers. I [cannot tell](https://github.com/esl/MongooseIM/pull/2929/commits/ef5eb332dabb12f21eb347c1602896de7aebe421) if you're gonna move that mod to TOML or drop it.

Since Cowboy natively uses maps HTTP headers, I'd like to map the headers as a map in the TOML. Will that work, even if I stray from the extra headers convention in `mod_revproxy`?

Reply:
The whole `mod_revproxy` functionality is deprecated and will be dropped in the next release unless a new demand appears (which is unlikely). Custom headers would need to be added to TOML as an table (which is parsed as a map by `tomerl`). However, the whole TOML parser is being rewritten at the moment and this code would need to be rewritten again.

Reply:
I'll wait until https://github.com/esl/MongooseIM/pull/2929 is merged. I don't think it'd make sense to pollute that branch with this change, which is more functionality than config. I also plan on adding some default security headers like: 

```erlang
{ws_https_custom_headers, "[[listen.http.handlers.mod_websockets.custom_headers]]
  strict-transport-security: "max-age=31536000 ; includeSubDomains"
"}.
```

to the HTTPS endpoints as part of the standard as part of the standard `vars-tool.config.in`.